rever/activities/push_tag.xsh

```diff
@@ -0,0 +1,59 @@
+"""Activity for pushing remote tags."""
+import re
+
+from rever import vcsutils
+from rever.activity import Activity
+from rever.tools import eval_version
+
+
+class PushTag(Activity):
+    """Pushes the current tag up to a remote repository.
+
+    This activity takes the following parameters:
+
+    :$TAG_REMOTE: str or None, remote URL to push tags to.
+    :$TAG_TARGET: str or None, remote branch to push to once the tag has been made.
```

Comment:
These are incorrect, sinc ethe name changed.  Should be `$PUSH_TAG_REMOTE` and `$PUSH_TAG_TARGET`.

Reply:
Also, there seem to be `PUSH_TAG_ORG` and `PUSH_TAG_REPO`